src/globals.cpp

```diff
@@ -25,6 +25,7 @@
 #include "primitives/block.h"
 #include "requestManager.h"
 #include "rpc/server.h"
+#include "script/standard.h"
```

Comment:
why this is needed?

Reply:
Its needed for the size of the OP_RETURN which I made a CTwek and is located in:
extern unsigned nMaxDatacarrierBytes;

Reply:
got it